### PR TITLE
Migration to 14.0 and several fixes

### PR DESCRIPTION
**Pull request by alegallo**

### CODE REVIEW

File: qms/models/finding.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def _stage_groups(self, stages, domain, order):
         comodel_name="qms.interested_party", required=True
     )
 
-    reference = fields.Char(required=True, readonly=True, default="NEW")
+    reference = fields.Char(required=True, readonly=False, default="NEW")
```
Comment (lisandrogallo) [inline qms/models/finding.py]: You can remove the 'readonly' property.

File: qms/models/indicator.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 from odoo import _, api, fields, models
-
+# from .indicator_measurement import IndicatorMeasurement
```
Comment (lisandrogallo) [inline qms/models/indicator.py]: You can remove this comment.

**Review by lisandrogallo — CHANGES_REQUESTED** [review]
Please, make this few changes before merge.

File: qms/models/finding.py (hunk 1 above)
Comment (alegallo) [reply qms/models/finding.py]: resolved

File: qms/models/indicator.py (hunk 2 above)
Comment (alegallo) [reply qms/models/indicator.py]: resolved

**Review by lisandrogallo — APPROVED** [review]

**Pull request merged**